crates/sui-types/src/crypto.rs

```diff
@@ -1625,7 +1625,9 @@ pub mod bcs_signable_test {
     }
 }
 
-#[derive(Deserialize, Serialize, JsonSchema, Debug, EnumString, strum_macros::Display)]
+#[derive(
+    Clone, Copy, Deserialize, Serialize, JsonSchema, Debug, EnumString, strum_macros::Display,
```

Comment:
Why we need Clone + Copy here?

Reply:
There was a change in clap in v4 that required types that it parses to impl Clone